### ISSUE TITLE
core: Size-based replica rebalancing

### ISSUE THREAD

**Issue by jseldess** [cockroachdb/docs#1560]
Background:
 - https://airtable.com/tblD3oZPLJgGhCmch/viwdV1sM5KEK2L14K/recYym8R1amHOYDoV
 - RFC: https://github.com/cockroachdb/cockroach/blob/master/docs/RFCS/rebalancing_for_1_1.md

PRs: 
- https://github.com/cockroachdb/cockroach/pull/16878

**Comment by jseldess** [comment]
@a-robinson, once this is ready for docs, please add the `ready` label and add links to relevant PRs.

**Comment by jseldess** [comment]
@a-robinson, I just noticed https://github.com/cockroachdb/cockroach/pull/16575. Does that mean this topic is ready to be documented for 1.1?

**Comment by a-robinson** [comment]
Nope, not yet. Still some work left to do.

**Comment by jseldess** [comment]
@a-robinson, I noticed that this is marked as "done" in AirTable. Also noticed the PR on "Rebalance replicas based on disk usage and write load." Marking this as 1.1 docs `ready`. Please remove and comment if you disagree.

In any case, corresponding docs depend on @sploiselle's upcoming architecture docs. Sean, this would be a 1.1-only update to the part on rebalancing.

**Comment by a-robinson** [comment]
Yup, docs for this can get started anytime.

**Comment by jseldess** [comment]
Off by default in 1.1. To be documented for 1.2.

**Comment by jseldess** [comment]
Closing in favor of https://github.com/cockroachdb/docs/issues/2051.
